src/Core/src/Platform/Windows/WrapperView.cs

```diff
@@ -130,12 +130,12 @@ void UpdateBorder()
 			_borderPath?.UpdateStrokeLineJoin(Border.StrokeLineJoin);
 		}
 
-		async partial void ShadowChanged()
+		partial void ShadowChanged()
 		{
 			if (HasShadow)
-				UpdateShadow();
+				UpdateShadowAsync().FireAndForget(IPlatformApplication.Current?.Services?.CreateLogger(nameof(WrapperView)));
```

Comment:
Looks somewhat hard to read. Can this be put on multiple lines?

Reply:
Question out of curiosity: could this not have been `partial async Task ShadowChangedAsync`?